pdoc/doc.py

```diff
@@ -82,6 +83,17 @@ def _is_exported(ident_name):
     return not ident_name.startswith("_")
 
 
+def _is_static(cls: typing.Type, method_name: str) -> bool:
+    """
+    Returns `True` if the given method is a @staticmethod.
+    """
+    for c in inspect.getmro(cls):
+        if method_name in c.__dict__:
+            return isinstance(c.__dict__[method_name], staticmethod)
```

Comment:
I don't think this check is sufficient. We use `classmethod` and others too to get things at class level.